package.json

```diff
@@ -60,6 +60,7 @@
     "remove-accents": "^0.4.2",
     "reselect": "^4.0.0",
     "sugarss": "^2.0.0",
+    "throttle-debounce": "^2.1.0",
```

Comment:
Since you already have lodash installed, you can use it instead of adding a new dependency: https://lodash.com/docs/4.17.11#throttle